README.md

```diff
@@ -15,13 +15,13 @@
 Project source code: [https://github.com/jarviscanada/jarvis_data_eng_GozdeArslan](https://github.com/jarviscanada/jarvis_data_eng_GozdeArslan)
 
 
-**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_GozdeArslan/tree/master/linux_sql)]: The Linux cluster Monitoring agent project contains solutions in the creation of infrastructure for the Jarvis Linux Cluster Administration(LCA) team in regards to server host monitoring. This infrastructure assists the team in examining host statistics and data usage. The monitoring agent servers will run on centOS 7 and will save host data (hardware specifications and resource use data) into the (RDBMS) Postgres Database every minute(Crontab). The LCA (Jarvis Linux Cluster Administration) team will be able to help servers in need by having records of each node and node monitoring systems. Technologies used in this project ;Bash Scripts,Docker,PostgresSQL Database,Git / Github, IntelliJ IDEA.
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_GozdeArslan/tree/master/linux_sql)]: Provided  solutions in the creation of infrastructure for the Jarvis Linux Cluster Administration(LCA) team in regards to server host monitoring. This infrastructure assists the team in examining host statistics and data usage. The monitoring agent servers will run on centOS 7 and will save host data (hardware specifications and resource use data) into the (RDBMS) Postgres Database every minute(Crontab). The LCA (Jarvis Linux Cluster Administration) team will be able to help servers in need by having records of each node and node monitoring systems 
```

Comment:
Developed a Linux cluster hardware usage monitoring agent using Bash scripts, PostgreSQL, Docker ,etc. 

Cronjob, SQL, 

https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee

Reply:
have you fixed this??? 
![image](https://user-images.githubusercontent.com/58951265/149435691-e6bdb699-3377-4f1f-b4be-6f5addb27e18.png)